cmd/detectExecuteScan.go

```diff
@@ -138,6 +138,14 @@ func detectExecuteScan(config detectExecuteScanOptions, _ *telemetry.CustomData,
 	if err != nil {
 		log.Entry().WithError(err).Warning("Failed to get GitHub client")
 	}
+
+	if config.BuildTool == golangBuildTool {
```

Comment:
Maybe is also good to check if config.PrivateModules is not empty.

Reply:
inside the function, the presence of an empty value is checked.